browser/ui/color/brave_color_mixer.cc

```diff
@@ -743,6 +743,37 @@ void AddBraveOmniboxColorMixer(ui::ColorProvider* provider,
 
   // We don't use bg color for location icon view.
   mixer[kColorOmniboxIconBackground] = {SK_ColorTRANSPARENT};
+
+#if defined(TOOLKIT_VIEWS)
+  if (!base::FeatureList::IsEnabled(
+          darker_theme::features::kBraveDarkerTheme) ||
+      key.scheme_variant != ui::ColorProviderKey::SchemeVariant::kDarker) {
+    return;
+  }
+
+  auto& postprocessing_mixer = provider->AddPostprocessingMixer();
+  // Location bar
+  postprocessing_mixer[kColorLocationBarBackground] =
+      darker_theme::CreateDarkerThemeColorTransform(
+          nala::kColorPrimitiveNeutral10);
+  postprocessing_mixer[kColorLocationBarBackgroundHovered] =
+      darker_theme::CreateDarkerThemeColorTransform(
+          nala::kColorPrimitiveNeutral10);
+
+  // Omnibox
+  postprocessing_mixer[kColorOmniboxResultsBackground] =
+      darker_theme::CreateDarkerThemeColorTransform(
+          nala::kColorPrimitiveNeutral10);
+  postprocessing_mixer[kColorOmniboxResultsBackgroundHovered] =
+      darker_theme::CreateDarkerThemeColorTransform(
+          nala::kColorPrimitiveNeutral20);
+  postprocessing_mixer[kColorOmniboxResultsBackgroundSelected] =
+      darker_theme::CreateDarkerThemeColorTransform(
+          nala::kColorPrimitiveNeutral20);
+  postprocessing_mixer[kColorBraveOmniboxResultViewSeparator] =
+      darker_theme::CreateDarkerThemeColorTransform(
+          nala::kColorPrimitiveNeutral20);
```

Comment:
Hey do you mind explaining how this works for me? So we take the lightness of the `kPrimitiveNeutralXXX` and apply it to whatever color we already had for that token?

Reply:
That'd be better. Will be on it as a folowup.

Reply:
Filed an issue to track it https://github.com/brave/brave-browser/issues/49492